frameworks/Go/go-raw/src/hello/hello.go

```diff
@@ -34,18 +34,7 @@ type Fortune struct {
 
 // Databases
 const (
-	// Go 1.4's sql.DB has scalability problem when using (explicitly reused) prepared statement.
-	// https://github.com/golang/go/issues/9484
-	//
-	// Using db.Query() instead of stmt.Query() avoid the issue.
-	// But it makes 3 round trips per query: prepare, execute and close.
-	// `interpolateParams=true` enables client side parameter interpolation.
-	// It reduces round trips without prepared statement.
-	//
-	// Before Go 1.5 is released, we can see real power of Go with this benchmark.
-	// After Go 1.5 is released, we can see prepared statement vs interpolation by comparing
-	// this and another lightweight Go framework.
```

Comment:
I've read this comment however I'd like to ask how important this comparison is given that:

- there are other differences between frameworks, so it is impossible to explain the difference in numbers by a single factor (prepared vs unprepared statements)
- scalability of prepared statements has been fixed in Go 1.5
- a version with prepared statements has the same or better performance with Go 1.5 and mysql
- using prepared statements is more secure and generally should be prefered in real production usage